### ISSUE TITLE
Video trimming: Uncaught promise

### ISSUE THREAD

**Issue by spacedmonkey** [swissspidy/web-stories-wp#11072]
## Bug Description

While testing video trimming, I noticed a  Uncaught promise in the console. See recording.. 

## Screenshots
https://user-images.githubusercontent.com/237508/159985250-932d807b-1922-4d02-9f9b-6bd9cfc85cf9.mov

## Additional Context

- Plugin Version:
- WordPress Version:
- Operating System:
- Browser:

**Comment by spacedmonkey** [comment]
Related: https://github.com/GoogleForCreators/web-stories-wp/issues/11073

**Comment by swissspidy** [comment]
Are these all videos that you recorded using the webcam recording POC? It seems like something is wrong with them. See how the duration is displayed as `NaN` everywhere and how the video length is missing in the media library.

Looks like what was inserted on the canvas by the webcam recorder is still the blob from the webcam stream that hasn't been replaced. So basically the video trimming tries to seek on an infinite (live)stream.

In short, this looks like something that's broken in the media recording in the first place, not video trimming or `seekVideo`
